src/ui.cpp

```diff
@@ -742,7 +747,7 @@ void uilist::show( ui_adaptor &ui )
     int estart = 1;
     if( !textformatted.empty() ) {
         for( int i = 0; i < text_lines; i++ ) {
-            trim_and_print( window, point( 2, 1 + i ), getmaxx( window ) - 4,
+            trim_and_print( window, point( 2, 1 + i ), getmaxx( window ) - WIDTH_BUFFER,
```

Comment:
`text` starts one space after the left border and end one space before the right border, so no need to change this.

Reply:
resolved by https://github.com/CleverRaven/Cataclysm-DDA/pull/64478/commits/2fbc662ed6e32eee9b009acc12f23fd661beb2c0